drivers/wifi/simplelink/simplelink_sockets.h

```diff
@@ -12,6 +12,7 @@ extern "C" {
 #endif
 
 extern const struct socket_offload simplelink_ops;
+extern void simplelink_sockets_init(void);
```

Comment:
Is `extern` intended here?

Reply:
yes, since the function is being called from simplelink.c: external linkage.
I understand extern is implicit on function declarations; but, well ... force of habit ;-)